### PR TITLE
<#40> Fix wrong attribute in processdualstart!()

### PR DESCRIPTION
**Pull request by migarstka**
- Change MOI.VariablePrimalStart to MOI.ConstraintDualStart in processdualstart!()

### CODE REVIEW

**Comment by blegat** [comment]
Could you add a test ?

File: test/MOI_wrapper.jl [hunk 1]
```diff
@@ -385,6 +385,49 @@ end
...
+    # define QP:
+    # min 1/2 x'Px + q'x
+    # s.t. Ax <= u   <=> -Ax + u in Nonnegatives
+    #      Ax >= l   <=> Ax - l in Nonnegatives
+
+    l = [1.; 0; 0];
+    u = [1.; 0.7; 0.7]
+
+    model = MOIU.UniversalFallback(OSQPModel{Float64}());
+    MOI.supports(::MOIU.UniversalFallback{OSQPModel{Float64}}, a::MOI.VariablePrimalStart, ::Type{MOI.VariableIndex}) = true
```
Comment (blegat) [inline test/MOI_wrapper.jl]: You shouldn't have to define that. The UniversallFallback supports everything

Comment (migarstka) [reply test/MOI_wrapper.jl]: Yes, the line is redundant. It doesn't cause the error though.
I get:
````
Got exception outside of a @test
  MathOptInterface.SetAttributeNotAllowed{MathOptInterface.VariablePrimalStart}: Setting attribute MathOptInterface.VariablePrimalStart() cannot be performed. You may want to use a `CachingOptimizer` in `AUTOMATIC` mode or you may need to call `resetoptimizer!` before doing this operation if the `CachingOptimizer` is in `MANUAL` mode.
  Stacktrace:
   [1] #throw_set_error_fallback#13(::MathOptInterface.SetAttributeNotAllowed{MathOptInterface.VariablePrimalStart}, ::Function, ::OSQPModel{Float64}, ::MathOptInterface.VariablePrimalStart, ::MathOptInterface.VariableIndex, ::Float64) at /Users/Micha/.julia/packages/MathOptInterface/o3zZH/src/attributes.jl:312
   [2] throw_set_error_fallback(::OSQPModel{Float64}, ::MathOptInterface.VariablePrimalStart, ::MathOptInterface.VariableIndex, ::Float64) at /Users/Micha/.julia/packages/MathOptInterface/o3zZH/src/attributes.jl:311
   [3] set(::OSQPModel{Float64}, ::MathOptInterface.VariablePrimalStart, ::MathOptInterface.VariableIndex, ::Float64) at /Users/Micha/.julia/packages/MathOptInterface/o3zZH/src/attributes.jl:289
   [4] set(::MathOptInterface.Utilities.UniversalFallback{OSQPModel{Float64}}, ::MathOptInterface.VariablePrimalStart, ::MathOptInterface.VariableIndex, ::Float64) at /Users/Micha/.julia/packages/MathOptInterface/o3zZH/src/Utilities/universalfallback.jl:270
   [5] _broadcast_getindex_evalf at ./broadcast.jl:574 [inlined]
   [6] _broadcast_getindex at ./broadcast.jl:547 [inlined]
   [7] getindex at ./broadcast.jl:507 [inlined]
   [8] copy at ./broadcast.jl:782 [inlined]
   [9] materialize(::Base.Broadcast.Broadcasted{Base.Broadcast.DefaultArrayStyle{1},Nothing,typeof(MathOptInterface.set),Tuple{Base.RefValue{MathOptInterface.Utilities.UniversalFallback{OSQPModel{Float64}}},Base.RefValue{MathOptInterface.VariablePrimalStart},Array{MathOptInterface.VariableIndex,1},Array{Float64,1}}}) at ./broadcast.jl:748
   [10] macro expansion at /Users/Micha/Dropbox/Research/Forks/OSQP.jl/test/MOI_wrapper.jl:422 [inlined]
   [11] macro expansion at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/Test/src/Test.jl:1083 [inlined]
   [12] top-level scope at /Users/Micha/Dropbox/Research/Forks/OSQP.jl/test/MOI_wrapper.jl:394
   [13] include at ./boot.jl:317 [inlined]
   [14] include_relative(::Module, ::String) at ./loading.jl:1041
   [15] include(::Module, ::String) at ./sysimg.jl:29
   [16] include(::String) at ./client.jl:388
   [17] top-level scope at none:0
   [18] eval(::Module, ::Any) at ./boot.jl:319
   [19] eval_user_input(::Any, ::REPL.REPLBackend) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.0/REPL/src/REPL.jl:85
   [20] run_backend(::REPL.REPLBackend) at /Users/Micha/.julia/packages/Revise/TmjcT/src/Revise.jl:766
   [21] (::getfield(Revise, Symbol("##58#60")){REPL.REPLBackend})() at ./task.jl:259
```

Comment (migarstka) [reply test/MOI_wrapper.jl]: The problem was a missing `MOI.` in front of one of the `get()`functions. Tests should pass now!

**Comment by migarstka** [comment]
@blegat I think someone has to trigger a rebuild on travis since the unit test for Julia v0.6 fails. However, this is unrelated to this PR since it happens when comparing a cold- and a warm-started solve time of an order of `1e-5` in the ` problem modification` testset. 
Because of the underterministic behaviour of that test it might be better if it gets removed. Maybe one could compare the number of iterations to convergence instead.

**Comment by tkoolen** [comment]
Restarted.

**Pull request merged**